### DISCUSSION TITLE
Flashing Firmware for Pixel 7a to Facilitate "SMS over Wi-Fi" in EMEA

### DISCUSSION THREAD

**Discussion by rybshik** [kyujin-cho/pixel-volte-patch#312] [Q&A]
Hello everyone,

I'm planning to install "Pixel IMS" on my Pixel 7a to enable "Wi-Fi Calling" and, most importantly, ***"SMS over Wi-Fi"*** with the Russian mobile provider MTS ( [MTS on Wikipedia](https://en.wikipedia.org/wiki/MTS_(telecommunications)) )

Before I proceed, I’m wondering if it would be beneficial to change the phone's factory firmware to optimize or enable "SMS over Wi-Fi" specifically for use in that region (EMEA?) 

For some context, the Pixel 7a was purchased in Canada, and it seems the current firmware is tailored for Canadian operators. 
***I’ll be using the device in a location where MTS doesn’t offer any cellular roaming, so all connectivity will be entirely via Wi-Fi.***

I’ve found factory images for the Pixel 7a on the [Google Developers website](https://developers.google.com/android/images#lynx), including options for different regions and Android versions. I’m considering flashing version 13.0.0 (TD4A.221205.042.A1, May 2023, **EMEA**) but I'm unsure if this is the best option. 

Here are the firmware versions I’m considering:
> **"lynx" for Pixel 7a**
> - 13.0.0 (TD4A.221205.042.A1, May 2023, **EMEA**)
> - 13.0.0 (TQ3A.230901.001, Sep 2023)
> - 14.0.0 (UP1A.231005.007.A1, Oct 2023, Japan)
> - 14.0.0 (UP1A.231105.003.A1, Nov 2023, JP carriers)
> - 14.0.0 (UQ1A.240205.002.B1, Feb 2024, Softbank)
> - 14.0.0 (AP2A.240805.005, Aug 2024)

Would flashing one of these versions optimize the device for SMS over Wi-Fi with MTS, or should I keep the original firmware, which updates automatically as needed?

Any advice or suggestions would be greatly appreciated!

**Comment by eliotcougar** [comment]
Pixel 9 pro here... I wasn't able to enable VoWiFi on MTS RUS for my phone... VoLTE works though...
